MolecularNodes/md.py

```diff
@@ -86,11 +86,34 @@ def load_trajectory(file_top,
     # separate the trajectory, separate to the topology or the subsequence selections
     traj = univ.trajectory[md_start:md_end:md_step]
     
+
     # if there is a non-blank selection, apply the selection text to the universe for 
     # later use. This also affects the trajectory, even though it has been separated earlier
     if selection != "":
         try:
-            univ = univ.select_atoms(selection)
+            # univ = univ.select_atoms(selection)
+            from solvation_analysis.solute import Solute
```

Comment:
This indeed works with his demo. With the bond fix, it will show the shell using Ball+Stick

Reply:
Ok, please check the bugfix submission I have in the discussion's page. I tried every test case with and without the bug fix. I was able to reproduce all your results. I wish I can credit you in the bug fix like submit it together.